pkgs/development/compilers/cmdstan/default.nix

```diff
@@ -44,7 +58,6 @@ stdenv.mkDerivation rec {
   preFixup = "rm -rf $(pwd)";
```

Comment:
That looks dangerous and if we are in the wrong directory could just delete everything